modules/ossm-installing-primary-remote-single-network-mesh.adoc

```diff
@@ -0,0 +1,138 @@
+// This procedure is used in the following assembly:
+// * install/ossm-multi-cluster-topologies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-installing-primary-remote-single-network-mesh_{context}"]
+= Installing a primary-remote single-network mesh 
+
+Install {istio} in a primary-remote single-network topology on two {ocp-product-title} clusters. 
+
+[NOTE]
+====
+In this procedure, `CLUSTER1` is the East cluster and `CLUSTER2` is the West cluster. 
+====
+
+You can adapt these instructions for a mesh spanning more than two clusters.
+
+.Prerequisites
+
+* You have installed the {SMProduct} Operator on all of the clusters that comprise the mesh.
```

Comment:
Same comment as in the other single-network doc. There must be direct connectivity between the pods in all clusters.